zoo-css-challenge/README.md

```diff
@@ -10,17 +10,13 @@ You are encouraged to use Google to help you find the answers to these problems.
 
 First of all, we don’t want the logo to have a purple border. The logo has an ID of logo, make sure this has no border.
 
-The first section has a class name of ‘introduction’.
-
-Give this section a white background.
-
-Add a paragraph to this section with the following text:
+The first section has a class name of ‘introduction’. Give this section a white background. Add a paragraph to this section with the following text:
 
 ‘The zoo is open every day of the year and features three major biomes: the Tropic Zone, Temperate Territory, and the Polar Circle. From tropical birds, to snow leopards, grizzly bears, and one of the nation’s largest colonies of Antarctic penguins, there are animals to enjoy in every season.’
 
-## 2) Bears:
+Most of the text on the page is very close together. Increase the amount of space of the text so that there's more breathing space.
```

Comment:
This was moved from `2) Bears`. It seems like the text that's close together isn't an issue with paragraph margin, but with text line height. I've updated the task to try and get the trainee to update `line-height`.